laletterbuilder/schema.py

```diff
@@ -33,9 +37,35 @@ def update_onboarding_info(cls, form, info: Dict[str, Any]):
 
     @classmethod
     def perform_post_onboarding(cls, form, request: HttpRequest, user: JustfixUser):
-        # TODO: Send SMS.
+        user.send_sms_async(
+            f"Welcome to {site_util.get_site_name()}, {user.best_first_name}! "
+            f"We'll be sending you notifications from this phone number.",
+        )
         pass
 
+    @classmethod
```

Comment:
Add a function comment here: "overloading BaseCreateAccounts to get email from this form rather than an earlier step"